application/models/Question.php

```diff
@@ -1565,20 +1563,11 @@ public function questionThemeNameValidator()
      * Returns the QuestionTheme related to this question.
      * It's not implemented as a relation because relations only work on
      * persisted models.
-     * It also returns the proper theme when 'question_theme_name' is empty or 'core'.
      *
      * @return QuestionTheme|null
      */
     public function getQuestionTheme()
     {
-        $questionThemeName = $this->question_theme_name;
```

Comment:
Some extra comments about this last commit.

- This method used to do more than returning the related theme. Also used to derived it in some specific cases where the question_theme_names was not explicit (core, or empty question_theme_names). This is not done anymore. Just as a side note.